src/views/Lottery/components/HowToPlay.tsx

```diff
@@ -338,8 +338,8 @@ const HowToPlay: React.FC = () => {
       </GappedFlex>
       <Divider />
       <Flex justifyContent="center" alignItems="center" flexDirection={['column', 'column', 'row']}>
-        <Flex maxWidth="240px" mr="8px" mb="16px">
-          <img src="/images/lottery/tombola.png" alt="tombola bunny" />
+        <Flex mr="8px" mb="16px">
+          <img style={{ width: '240px', height: '172px' }} src="/images/lottery/tombola.png" alt="tombola bunny" />
```

Comment:
Can you use the `Image` component instead?

Reply:
Done